### DISCUSSION TITLE
Extract correct changelog for ansible python package

### DISCUSSION THREAD

**Discussion by Yannik** [renovatebot/renovate#30224] [Suggest an Idea]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab, latest

### Please tell us more about your question or problem

I have a pip requirement for the [ansible](https://pypi.org/project/ansible/) package. This package does get upgrade PRs from renovate. Unfortunately, the changelog is not shown/linked for these package upgrades. It would be tremendously helpful to be able to quickly review the changelog to determine if the suggested upgrade can be merged safely.

The ansible package does not provide github releases, but there is a changelog file in the repository. This changelog file exists per-major-version: https://github.com/ansible-community/ansible-build-data/blob/main/10/CHANGELOG-v10.md#v10-0-0 and contains an anchor to the relevant minor/patch version.

It would be awesome if renovate could link the relevant changelog.

It does not seem possible to use [changelogUrl](https://docs.renovatebot.com/configuration-options/#changelogurl), since it seem like this only supports a static link, and not one based on the version of the release.

**Comment by RahulGautamSingh** [comment]
Sorry, the bot doesn't yet support such cases.

**Comment by Yannik** [comment]
@RahulGautamSingh I know it doesn't right now... Let's treat this as a feature request.

**Reply by RahulGautamSingh** [reply]
Discussion Category: should be Suggest an Idea then :)

And, let's wait for a maintainer's review.

**Reply by Yannik** [reply]
You are right. Done!
